modules/manage/partials/schema-reg-ui.adoc

```diff
@@ -0,0 +1,117 @@
+In the Redpanda UI, the **Schema Registry** tab lists registered and verified schemas, including their serialization format and versions. Select an individual schema to see which topics it applies to. 
+
+[NOTE]
+====
+The Schema Registry is built into Redpanda, and you can use it with the Schema Registry API or with the Redpanda UI. This section describes Schema Registry operations available in the Redpanda UI. See also:
+
+* xref:manage:schema-reg/schema-reg-overview.adoc[]
+* xref:manage:schema-reg/schema-reg-api.adoc[]
+====
+
+ifdef::env-console[]
```

Comment:
I wouldn't use Console as the environment. Instead we want this to be displayed in all contexts that aren't Redpanda Cloud, so:

```asciidoc
ifndef::env-cloud[]
== Prerequisites

Redpanda Console is automatically configured with Redpanda Cloud deployments. However, if you're running Redpanda Console with self-hosted Redpanda, you must add a valid `schemaRegistry` configuration in Redpanda Console:

.redpanda-console-config.yaml
[,yaml]
----
kafka:
  schemaRegistry:
    enabled: true
    urls: ["https://my-schema-registry.com"]
    username: console
    password: redacted # Or set using flags or env variable
  # To enable protobuf support
  protobuf:
    enabled: true
    schemaRegistry:
      enabled: true
      refreshInterval: 5m
----
endif::[]
```